pkg/cmd/agent-task/list/list.go

```
@@ -73,9 +83,24 @@ func listRun(opts *ListOptions) error {
 
 	opts.IO.StartProgressIndicatorWithLabel("Fetching agent tasks...")
 	defer opts.IO.StopProgressIndicator()
-	sessions, err := capiClient.ListSessionsForViewer(context.Background(), opts.Limit)
-	if err != nil {
-		return err
+	var sessions []*capi.Session
+	ctx := context.Background()
+
+	var repo ghrepo.Interface
+	if opts.BaseRepo != nil {
+		repo, _ = opts.BaseRepo()
+	}
```

Why are we swallowing the repo resolution error?

---

Per our sync, I added a comment in 227f0bd01d7ca712f2bccb8644a6dd930d7376b3